### PR TITLE
Project Survey - Felicia Liabäck Löwstett

### PR DESCRIPTION
**Pull request by liaback-lowstett**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/Components/Summary.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <section className="summary-container">
+        <p>
+            Thank you <span className="answer-text">{name} </span> for taking your time and filling out the survey! 
+            Here are a summary of your answers: 
+        </p>
+        <p>
+            It seems like you are <span className="answer-text">{evaluation}</span> to become a developer.
+        </p>
+        <p>
+            You paragraph project was the <span className="answer-text">{project}</span>.
```
Comment (Sofarz) [inline code/src/Components/Summary.js:15]: You probably mean your favourite :)

**Review by Sofarz — APPROVED** [review]
Really good job on the survey project! Way to go on using useEffect and useCallback before we even started going through this :).

The website looks nice and clean, however adding a restart button in the summary would have been a nice touch in case the user would want to fill in your survey over and over again :).

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Please add the Netlify link so I can test it 😸

**Comment by Sofarz** [comment]
https://youthful-almeida-30dc8d.netlify.app/

File: code/src/Components/Form.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        name={name}
+        evaluation={evaluation}
+        project={project}
+        thoughts={thoughts}
+      />
+    );
+  }
+
+  return (
+    <form
```
Comment (KarolCislowski) [inline code/src/Components/Form.js:61]: You should have only one return section inside of your component. You can achieve the same effect with a structure like this:
``` jsx 
  return (
    <>
       { submitted ? 
           <Summary />
             :
           <form>...
        }
    </>
  )
```

**Review by KarolCislowski — APPROVED** [review]
Great job!
Maybe for the next step, you will try to add some more exotic types of inputs? 🍹

**Pull request closed without merging**